device-types/Intel/H2312XXKR2.yaml

```diff
@@ -0,0 +1,32 @@
+---
+manufacturer: Intel
+model: H2312XXKR2
+slug: intel-h2312xxkr2
+u_height: 2
+is_full_depth: true
+comments: |
+  [H2312XXKR2 Specifications on intel.com](https://www.intel.com/content/www/us/en/products/sku/81877/intel-server-chassis-h2312xxkr2/specifications.html)
+subdevice_role: parent
+device-bays:
+  - name: Node 1
+    label: Blade 1
+  - name: Node 2
+    label: Blade 2
+  - name: Node 3
+    label: Blade 3
+  - name: Node 4
+    label: Blade 4
+power-ports:
```

Comment:
Just as in the other one, the PSUs are modular. This definition should be removed and `module-bays` should be created
Then create the applicable PSUs in the module-types folder

Looks like the PSUs are `FXX1600PCRPS`